client/components/popover/util.js

```diff
@@ -38,42 +38,45 @@ const adjacent = {
 	right: 'bottom',
 };
 
-//let viewport;
+let _viewport = null;
+let _windowEventsRefCount = 0;
 
-let viewport = updateViewport();
+function getViewport() {
+	if ( ! _viewport ) {
+		// initialize on first use
+		_viewport = updateViewport();
+	}
 
-function onViewportChange() {
-	viewport = updateViewport();
+	return _viewport;
 }
 
-let windowEventsBound = false;
+function onViewportChange() {
+	_viewport = updateViewport();
+}
 
 const bindWindowListeners = () => {
-	if ( windowEventsBound ) {
-		return debug( 'window events already bound' );
+	if ( _windowEventsRefCount++ > 0 ) {
```

Comment:
this may warrant a comment? You're always incrementing the ref count, even when it's greater than zero...

Reply:
Yes, the first popover that mounts installs the listeners, the last one that unmounts uninstalls them. Traditional reference counting. The counter tracks the number of active popovers.